src/eventmaster/api_handlers.go

```diff
@@ -98,8 +98,8 @@ func NewHTTPServer(tlsConfig *tls.Config, store *EventStore) *http.Server {
 
 	r.GET("/v1/health", wrapHandler(h.handleHealthCheck))
 
-	// GitHub webhook endpoint
-	r.POST("/v1/github_event", wrapHandler(h.handleGitHubEvent))
+	// plugins endpoint
+	r.POST("/v1/plugins/:name", wrapHandler(h.handlePluginEvent))
```

Comment:
Is the plan to support multiple "events" for plugins to handle? If so, it might be good to have this split out into separate things such as `/plugins/:name/add`